components-core/src/main/java/com/adyen/checkout/components/core/internal/util/CheckoutAttemptIdMapper.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by ararat on 8/10/2024.
+ */
+
+package com.adyen.checkout.components.core.internal.util
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.components.core.internal.analytics.AnalyticsManager
+import com.adyen.checkout.components.core.internal.analytics.CheckoutAttemptIdState
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+const val CHECKOUT_ATTEMPT_ID_NOT_FETCHED = "checkoutAttemptId-not-fetched"
```

Comment:
All these fields and methods should be internal or even moved to `DefaultAnalyticsManager` IMO. These two `val`s are now used in tests, but I think if you make the change I suggested earlier and hardcode a value in `TestAnalyticsManager` then tests don't have to know about these